frontend/public/components/namespace.jsx

```diff
@@ -83,10 +91,44 @@ import {
 import { removeQueryArgument } from './utils/router';
 import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
 
+import NamespaceDropdown from '../../packages/console-app/src/components/namespace/NamespaceDropdown';
+
+import {
+  isCurrentUser,
+  isOtherUser,
+  isSystemNamespace,
+} from '../../packages/console-app/src/components/namespace/filters';
+
 const getModel = (useProjects) => (useProjects ? ProjectModel : NamespaceModel);
 const getDisplayName = (obj) =>
   _.get(obj, ['metadata', 'annotations', 'openshift.io/display-name']);
-const CREATE_NEW_RESOURCE = '#CREATE_RESOURCE_ACTION#';
+
+export const REQUEST_FILTER = { me: 'me', user: 'user', system: 'system' };
```

Comment:
nit, a TypeScript enum would be a good fit here.

Reply:
Changed to enum